src/NServiceBus.Core/Licensing/LicenseManager.cs

```diff
@@ -100,6 +108,7 @@ static bool FirstTimeUser()
             var postData = $"version={version}";
             try
             {
+                // Call the web api. 
```

Comment:
looks like we could move the API call into it's dedicated method instead of the (rather useless) comment?